cmd/clusterctl/client/cluster/cert_manager_test.go

```diff
@@ -635,7 +674,7 @@ func Test_certManagerClient_PlanUpgrade(t *testing.T) {
 
 }
 
-func Test_certManagerClient_EnsureLatestVersion(t *testing.T) {
+func Test_certManagerClient_EnsureLaexpectedVersion(t *testing.T) {
```

Comment:
```suggestion
func Test_certManagerClient_EnsureLatestVersion(t *testing.T) {
```
Looks like the search and replace got this one too 🙂

Reply:
:) thanks for point out, I didn't notice this ..